templates/api/clients/ruby/client.gemspec.tpl

```diff
@@ -4,14 +4,18 @@ require_relative 'lib/{{ .Config.Name }}_client/version'
 Gem::Specification.new do |spec|
   spec.name          = "{{ .Config.Name }}_client"
   spec.version       = {{ title .Config.Name }}Client::VERSION
-  spec.summary       = "ruby client for {{ .Config.Name }} service"
+  {{- if stencil.Arg "service" }}
+  spec.summary       = "gRPC client for {{ .Config.Name }} service"
+  {{- else }}
+  spec.summary       = "gRPC types for {{ .Config.Name }}"
+  {{- end }}
   spec.authors       = ["{{ stencil.Arg "reportingTeam" }}"]
   spec.homepage      = "https://github.com/getoutreach/{{ .Config.Name }}"
   spec.metadata["homepage_uri"] = spec.homepage
   spec.metadata["source_code_uri"] = spec.homepage
   spec.metadata["github_repo"] = "ssh://github.com/getoutreach/{{ .Config.Name }}"
 
-  spec.required_ruby_version = Gem::Requirement.new(">= 2.7.5")
+  spec.required_ruby_version = Gem::Requirement.new(">= {{ stencil.Arg "versions.grpcClients.ruby" }}")
```

Comment:
Is it possible that the arg `versions.grpcClients.ruby` would be empty?

Reply:
I think if it is, it's a willful change by the owner of that repo:

https://github.com/getoutreach/stencil-golang/blob/816363413411a83c375a0a22adcd208d2ed1bb75/manifest.yaml#L192-L198

Reply:
I see, the default version was set so should be good.